test/functional/bip9-softforks.py

```diff
@@ -15,16 +15,17 @@
 test that enforcement has not triggered (which triggers ACTIVE)
 test that enforcement has triggered
 """
+from io import BytesIO
+import shutil
```

Comment:
Yes, this was being indirectly imported from utils.py :( there are still lots of indirect imports in functional tests, which makes refactors like this seem more disruptive than they actually are.